src/assets/data/map_data/auditoriums.ts

```diff
@@ -1,111 +1,112 @@
 import { GROUND_FLOOR_IMG_URL, FIRST_FLOOR_IMG_URL } from "./floor_images";
 import i18n from "../../../i18n";
 import elevator from "./elevator_positions";
-
-let locations;
-export default locations = [
+import { Location } from "../../../hooks/useMapData";
+const locations:Location[] = [
   {
-    floor: i18n.t("ground_floor"),
+    floor: i18n.t("ground_floor") as string,
```

Comment:
The result might be null and we're assigning `null` to a `string | undefined` value. 
Resolve it with `i18n.t(...) ?? undefined`

Reply:
missed the first one i think